tests/reftests/var-option.test

```diff
@@ -598,7 +598,7 @@ true
 true
 ### opam var arch="somearch" | '…' -> '...' | '`' -> "'"
 opam: variable setting needs a scope, use '--global' or '--switch <switch>'
-Usage: opam var [OPTION]... [VAR[=[VALUE]]]
+Usage: opam var [--global] [--package=PACKAGE] [OPTION]... [VAR[=[VALUE]]]
```

Comment:
We might want to add a new mandatory `~section` argument to all the argument functions to avoid this type of output, but it's not really a big deal

Reply:
fixed in #5275

Reply:
integrated here